### ISSUE TITLE
Organize samples as submodules of a single `:sample` gradle module

### ISSUE THREAD

**Issue by cortinico** [Yelp/swagger-gradle-codegen#14]
The more we keep on adding samples to this project (we now have 3 but they might grow), the more gradle modules this project will have. Gradle allows to create submodules so ideally we should have a structure like this:

```
├── build.gradle.kts
├── plugin
│   └── build.gradle.kts
├── samples
│   └── sample-groovy-android
|          └── build.gradle.kts
|          └── README.md
│   └── sample-kotlin-android
|          └── build.gradle.kts
|          └── README.md
│   └── ...
|          └── build.gradle.kts
|          └── README.md
└── settings.gradle.kts
```

Every module will have its own markdown file with samples and short guides on how to use it. The idea is to achieve something like this: https://github.com/gradle/kotlin-dsl/tree/master/samples

**Comment by MatthewPageCS** [comment]
@cortinico I'd be able to pick this up :)

(I don't see a way to assign the issue to myself, perhaps you need permissions to do so, it's been a while since I used github).

**Comment by cortinico** [comment]
Closed with #17
